wordpress/wp-content/plugins/memberful-wp/src/paywall/renderer.php

```
@@ -0,0 +1,346 @@
+	 *
+	 * @return string
+	 */
+	private static function render_simple( array $config ): string {
+		return '<div class="memberful-paywall__inner">'
+			. self::heading_block( $config )
+			. self::subheading_block( $config )
+			. self::features_block( $config )
+			. '<div class="memberful-paywall__actions">' . self::primary_cta( $config ) . '</div>'
+			. self::sign_in_prompt( $config )
+			. '</div>';
+	}
```

PHPMD (2.15.0)

[warning] Avoid unused private methods such as 'render_simple'. (undefined)

(UnusedPrivateMethod)